packages/frontend/src/components/SavedQueries/CreateSavedQueryModal.tsx

```diff
@@ -43,6 +44,19 @@ const CreateSavedQueryModal: FC<CreateSavedQueryModalProps> = ({
                             placeholder="eg. How many weekly active users do we have?"
                         />
                     </FormGroup>
+                    <FormGroup
+                        label="Chart description"
+                        labelFor="chart-description"
+                        style={{ fontWeight: 'bold' }}
```

Comment:
inline style ? :wink:

Reply:
😮  Good catch!! this was a copy paste job 🙈